tests/generate_visualisation_input.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+import re
+from types import SimpleNamespace
+from PopPUNK.assign import assign_query_hdf5
+from PopPUNK.web import summarise_clusters, sketch_to_hdf5
+from PopPUNK.utils import setupDBFuncs
+from beebop.filestore import PoppunkFileStore, DatabaseFileStore
+from tests import setup
+
+
+def hex_to_decimal(sketches_dict):
+    for sample in list(sketches_dict.values()):
+        if type(sample['14'][0]) == str and re.match('0x.*', sample['14'][0]):
+            for x in range(14, 30, 3):
+                sample[str(x)] = list(map(lambda x: int(x, 16),
+                                          sample[str(x)]))
+
+
+storageLocation = './tests/files'
+p_hash = 'unit_test_visualisations'
+fs = PoppunkFileStore(storageLocation)
+outdir = fs.output(p_hash)
+if not os.path.exists(outdir):
+    os.mkdir(outdir)
+db_paths = DatabaseFileStore('./storage/GPS_v4_references')
+with open("./beebop/resources/args.json") as a:
+    args_json = a.read()
+args = json.loads(args_json, object_hook=lambda d: SimpleNamespace(**d))
```

Comment:
this block of three lines turns up in three places, I suggest you pop it in a function and save yourself some typing :)